### ISSUE TITLE
Grav language API returns false

### ISSUE THREAD

**Issue by NicoHood** [getgrav/grav#3045]
I noticed, that unless you define any supported language the following commands will all return false. This is not documented and also confusing to me:
```
{{ grav.language.enabled|defined('no')|default('false') }}
{{ grav.language.getDefault }}
{{ grav.language.getActive }}
{{ grav.language.getLanguage }}
```

```yaml
languages:
  translations: true
  http_accept_language: true
  include_default_lang: false
  default_lang: en
  # supported:
  #   - en
```

I expect that the functions return `en` or if `http_accept_language` is used any other language.

Another issue I am facing is, that if I set supported languages (en, de) the browser now always redirects to `/de/index.php` for example, when I am using a german browser. If i do not specify and supported language it will show all languages under the same url. That behavior is inconsistent to me.

**Comment by mahagr** [comment]
Yup, they return false and I tried to fix that in Grav 1.7 but that broke too many things so I've left them as they were -- but added typehints into phpdocs to tell what they return.

**Comment by hughbris** [comment]
The first issue @NicoHood mentions here (`getDefault` returning `false` and `getActive` returning `null` when languages not explicitly set) caused an error in a plugin I wrote. That's because I wanted to translate a translatable string I defined in the plugin (with English only so far).

I think at the least, @NicoHood is right that the current behaviour is undocumented.

I've worked around my error by falling to back to 'en', but I feel like that should be handled by core. [There's precedent in core for this kind of fallback](https://github.com/getgrav/grav/blob/42084ea0cb5f3aebcb385a1a797af72000b6d1e1/system/src/Grav/Common/Language/Language.php#L421), but it's not applied everywhere.

If you think this is a new issue, let me know and I'll open it.
